vj4/model/domain.py

```diff
@@ -114,6 +137,17 @@ def get_multi(*, fields=None, **kwargs):
 
 
 @argmethod.wrap
+async def transfer(domain_id: str, old_owner_uid: int, new_owner_uid: int):
+  for domain in builtin.DOMAINS:
+    if domain['_id'] == domain_id:
+      raise error.BuiltinDomainError(domain_id)
+  coll = db.coll('domain')
+  return await coll.find_one_and_update(filter={'_id': domain_id, 'owner_uid': old_owner_uid},
+                                        update={'$set': {'owner_uid': new_owner_uid}},
+                                        return_document=ReturnDocument.AFTER)
```

Comment:
Set `root` role for new owner?

Reply:
root role is granted by default at creation. after that we don't take care of it anymore. domain_uid is only used to trace the creator of the domain.